app/src/modules/settings/routes/extensions/extensions.vue

```diff
@@ -79,6 +113,17 @@ fetchExtensions();
 				{{ t('no_extensions_copy') }}
 			</v-info>
 		</div>
+
+		<v-dialog v-model="confirmLeave" @esc="confirmLeave = false">
+			<v-card>
+				<v-card-title>{{ t('extension_reload_required') }}</v-card-title>
+				<v-card-text>{{ t('extension_reload_required_copy') }}</v-card-text>
+				<v-card-actions>
+					<v-button @click="leavePage">{{ t('extension_reload_now') }}</v-button>
+					<v-button secondary @click="confirmLeave = false">{{ t('back') }}</v-button>
+				</v-card-actions>
+			</v-card>
+		</v-dialog>
```

Comment:
Nice idea to add a navigation guard 🎉 

However, I'm not quite sure whether we really want to force the page reload, seems a bit too "aggressive" for me.
How about offering the option "Proceed without reload" instead of "Back"?

Also the primary action should be moved to the right side:
```suggestion
		<v-dialog v-model="confirmLeave" @esc="confirmLeave = false">
			<v-card>
				<v-card-title>{{ t('extension_reload_required') }}</v-card-title>
				<v-card-text>{{ t('extension_reload_required_copy') }}</v-card-text>
				<v-card-actions>
					<v-button secondary @click="confirmLeave = false">{{ t('back') }}</v-button>
					<v-button @click="leavePage">{{ t('extension_reload_now') }}</v-button>
				</v-card-actions>
			</v-card>
		</v-dialog>
```

Reply:
I personally think a reload should be required as the App is effectively in an error state after disabling extensions and in an incomplete state when enabling extensions. By enforcing this with the navigation guard i'm hoping to prevent issues like "I just enabled interface X but cannot select it when creating a field" or people selecting disabled interfaces which will be non-existing for anyone else or after reload.